packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -344,6 +346,33 @@ contract OptimismPortal_Test is CommonTest {
         vm.expectRevert(stdError.indexOOBError);
         assertEq(optimismPortal.isOutputFinalized(nextOutputIndex + 1), false);
     }
+
+    function testFuzz_setGasPayingToken_succeeds(
+        address _token,
+        uint8 _decimals,
+        bytes32 _name,
+        bytes32 _symbol
+    )
+        external
+    {
+        vm.expectEmit(address(optimismPortal));
+        emit TransactionDeposited(
+            0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001,
+            Predeploys.L1_BLOCK_ATTRIBUTES,
+            0,
+            abi.encodePacked(
+                uint256(0), // mint
+                uint256(0), // value
+                uint64(80000), // gasLimit
+                false, // isCreation,
+                abi.encodeCall(L1Block.setGasPayingToken, (_token, _decimals, _name, _symbol))
+            )
+        );
+
+        vm.prank(address(systemConfig));
+        optimismPortal.setGasPayingToken({ _token: _token, _decimals: _decimals, _name: _name, _symbol: _symbol });
+        //assertEq(optimismPortal.gasPayingToken(), _token);
```

Comment:
this line should be deleted, because implementation of `optimismPortal.setGasPayingToken()`doesn't update the token name (can you confirm if that's intended behavior?)

Reply:
Ah you are right, the source of truth on L1 is the system config